src/main/scala/org/apache/spark/sql/OapTestSuite.scala

```diff
@@ -113,6 +116,13 @@ abstract class OapTestSuite extends BenchmarkConfigSelector with OapPerfSuiteCon
   def resultMap = _resultMap
 
   protected def testSet: Seq[OapBenchmarkTest]
+  protected def dropCache(): Unit = {
+    val nodes = spark.sparkContext.getExecutorMemoryStatus.map(_._1.split(":")(0))
```

Comment:
This looks trivial, do we have an elegant way to do that?

Reply:
Yeah...kind of a ugly hack. I didn't come up with another idea. Will try to find out if there's a better way.